scripts/zkai_stwo_ai_two_head_seq32_layout_schedule_sweep_gate.py

```
@@ -0,0 +1,786 @@
+def blake2b_commitment(value: Any, domain: str) -> str:
+    hasher = hashlib.blake2b(digest_size=32, person=b"zkai-layout-v1")
+    hasher.update(domain.encode("utf-8"))
+    hasher.update(b"\0")
+    hasher.update(canonical_json_bytes(value))
+    return "blake2b-256:" + hasher.hexdigest()
+
+
+def sha256_file(path: pathlib.Path, expected_sha256: str | None = None) -> str:
+    digest = hashlib.sha256(path.read_bytes()).hexdigest()
+    if expected_sha256 is not None and digest != expected_sha256:
+        raise LayoutScheduleSweepGateError(f"{path.name} sha256 drift")
```

Ruff (0.15.15)

[warning] Avoid specifying long messages outside the exception class

(TRY003)